### PR TITLE
feat: added typography and lists components

### PR DESCRIPTION
**Pull request by keshavkumar-in**
Added Typography Component and Lists Component. Please give feedback.

### CODE REVIEW

File: docs/components/typography/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <h5>H5 Heading</h5>
+      <h4>H4 Heading</h4>
+      <h3>H3 Heading</h3>
+      <h2>H2 Heading</h2>
+      <h1>H1 Heading</h1>
+    </div>
+    <div>
+      <p class="h6">H6 Heading</p>
+      <p class="h5">H5 Heading</p>
+      <p class="h4">H4 Heading</p>
```
Comment (vrx29) [inline docs/components/typography/index.html:38]: You defined heading with heading tags as well as heading classes like h1,h2. I think you should not define heading in class name and should not use them in p tag. If you have few properties for your heading just apply them to heading tags directly like 
h1{
  font-size:4rem;
}
If you want to have classes for text your use classes like text-lg,text-sm like this .

**Review by vrx29 — APPROVED** [review]
your code is clean and well structured good to merge.

**Review by Dev-AkshatGupta — APPROVED** [review]
I feel everything is great except the thing Vineet mentioned

File: docs/components/typography/index.html (hunk 1 above)
Comment (govindappaarun) [reply docs/components/typography/index.html:38]: Commit name or description should be bit more clear, 
you are not adding components in this PR, but documentation only

**Review by govindappaarun — APPROVED** [review]

File: docs/components/typography/index.html (hunk 1 above)
Comment (keshavkumar-in) [reply docs/components/typography/index.html:38]: Sure, will remove that in refactoring.

**Pull request merged**